airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -95,6 +94,7 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     // and can't
     // remove zero date values.
     // since zero dates are placeholders, we convert them to null by default
+    //FIXME(girarda): do we also want to prevent people from overriding zeroDateTimeBehavior?
     jdbcUrl.append("?zeroDateTimeBehavior=convertToNull");
```

Comment:
@tuliren do we want to support overwriting `zeroDateTimeBehavior` ?

Reply:
No, for all the existing parameters added by us, let's prevent them from getting overwritten. The downstream data process relies on this `zeroDateTimeBehavior`. In the future this may change, but for now let's just hardcode it as is.

Reply:
Done.